README.md

```diff
@@ -4,15 +4,36 @@
+| spring.graphql.graphiql.enabled           | Enable GraphiQL web interface for GraphQL | true                                             | true                                                                 |
+| spring.graphql.graphiql.path              | Path for GraphiQL when enabled            | /graphiql                                        | /graphiql                                                            |
+| spring.profiles.active                    | Active Spring profile                     | dev                                              | prod                                                                 |
+| spring.jpa.properties.hibernate.dialect   | Hibernate dialect for PostgreSQL          | org.hibernate.dialect.PostgreSQLDialect          | org.hibernate.dialect.PostgreSQLDialect                              |
+| spring.datasource.url                     | PostgreSQL database URL                   | jdbc:postgresql://localhost:4032/content-service | jdbc:postgresql://content-service-db-postgresql:5432/content-service |
+| spring.datasource.username                | Database username                         | root                                             | gits                                                                 |
+| spring.datasource.password                | Database password                         | root                                             | ${db_password}                                                       |
+| spring.datasource.driver-class-name       | JDBC driver class                         | org.postgresql.Driver                            | org.postgresql.Driver                                                |
+| spring.sql.init.mode                      | SQL initialization mode                   | always                                           | always                                                               |
+| spring.jpa.show-sql                       | Show SQL queries in logs                  | true                                             | true                                                                 |
+| spring.sql.init.continue-on-error         | Continue on SQL init error                | true                                             | true                                                                 |
+| spring.jpa.hibernate.ddl-auto             | Hibernate DDL auto strategy               | create                                           | update                                                               |
+| hibernate.create_empty_composites.enabled | Enable empty composite types in Hibernate | true                                             | true                                                                 |
+| DAPR_GRPC_PORT                            | Dapr gRPC Port                            | 50001                                            | 50001                                                                |
```

Comment:
This is not set in DEV

Reply:
> This is not set in DEV

yes I already Removed this in my previous commit